### PR TITLE
added save mutation with separated components

### PR DESCRIPTION
**Pull request by ochanje210**

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -41,7 +43,7 @@ export function setNetworkLayer() {
   let options = {};
 
   // Access Token
-  const authToken = '';
+  const authToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjYxcmlPZmhVWFdTU3BQVHJOUWJrWGNrS1N3dDEiLCJlbWFpbCI6IjEyM0AxMjMuY29tIiwibmFtZSI6IuygnOyYgeywrCIsImlhdCI6MTQ4NDU2OTQyMn0.ROpx-5bQFAK2Q4bnogu9p2dr82KhZgD_-sXHFhNGI5w';
```
Comment (ace0625) [inline app/app.js]: delete the token

File: app/app.js [hunk 2]
```diff
@@ -75,6 +77,15 @@ export default class App extends React.Component {
           onPress={() => {
             this.setState({currentScene: text});
             Actions.refresh();
+            if (text === SCENE_CONSTANT.MY_LIBRARY) {
+              Actions.myLibrary({prevScene: text});
+            }
```
Comment (ace0625) [inline app/app.js]: change the if-else line spaces.

File: app/components/detailView/flipCard.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
-import React, { Component } from 'react';
+import React, { Component, PropTypes } from 'react';
 import FlipCard from 'react-native-flip-card';
 import DetailView from './detailView';
+import { SCENE_CONSTANT } from './../../app';
 
 export default class FlipCardDetailView extends Component {
   constructor() {
```
Comment (ace0625) [inline app/components/detailView/flipCard.js:7]: Seems constructor is not needed here.

File: app/components/myLibrary/recommended.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
+import React, { Component, PropTypes } from 'react';
+import { View } from 'react-native';
+import Relay from 'react-relay';
+import LibraryView from './libraryView';
+import { SCENE_CONSTANT } from './../../app';
+
+export class Recommended extends Component {
+  constructor(props) {
+    super(props);
+  }
```
Comment (ace0625) [inline app/components/myLibrary/recommended.js:10]: new line space.

File: app/components/myLibrary/saved.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
+import React, { Component, PropTypes } from 'react';
+import { View } from 'react-native';
+import Relay from 'react-relay';
+import LibraryView from './libraryView';
+import { SCENE_CONSTANT } from './../../app';
+
+export class Saved extends Component {
+  constructor(props) {
+    super(props);
+  }
```
Comment (ace0625) [inline app/components/myLibrary/saved.js:10]: new line space.

File: app/components/detailView/flipCard.js (hunk 3 above)
Comment (ochanje210) [reply app/components/detailView/flipCard.js:7]: I think you missed that state is being used here in constructor

File: app/components/myLibrary/recommended.js (hunk 4 above)
Comment (ochanje210) [reply app/components/myLibrary/recommended.js:10]: fixed

File: app/components/myLibrary/saved.js (hunk 5 above)
Comment (ochanje210) [reply app/components/myLibrary/saved.js:10]: fixed

**Comment by ace0625** [comment]
LGTM

[Approved with PullApprove](https://pullapprove.com/recbook/recbook-client/pull-request/31/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=recbook/recbook-client)

**Pull request merged**